NuGet.Docs/docs/Get-Started/Install-NuGet.md

```diff
@@ -1,13 +1,112 @@
+</div>
+
+
+####Try out some scenarios
+* [Finding and installing a package](https://docs.nuget.org/Consume/Package-Manager-Console#finding-a-package)</li>
+* [Removing a package](https://docs.nuget.org/Consume/Package-Manager-Console#removing-a-package)</li>
+* [Updating a Package](https://docs.nuget.org/Consume/Package-Manager-Console#updating-a-package)</li>
+
+
+####References
+* [Package Manager Console Powershell Commands](https://docs.nuget.org/consume/package-manager-console-powershell-reference)
+* [Setting up a NuGet Powershell Profile](https://docs.nuget.org/Consume/Setting-up-NuGet-PowerShell-Profile.md)</li>
+
+
+###Updating NuGet Package Manager extension in Visual Studio
+You can update NuGet using the Visual Studio Extension Manager. Navigate to the Extension Manager and click on the Updates tab to check for updates. If there is a new version of NuGet you will see it in the list of available updates. From VS 2015 Update 2, NuGet extension is auto updated by default in Visual Studio.
+
+##NuGet Beta Channel
+NuGet Beta Channel for the Visual Studio 2015 NuGet Package Manager gives you access to Beta bits. Even though it's called beta, we will only release near RTM quality builds into this channel. We want to use the extra runway on our end to make sure that we incorporate any feedback that we might get from the users of the channel and catch any blocking issues early on.
+
+<div class="block-callout-info">
+	Remember if you run into any issues while dogfooding the beta build or have an idea, <a href="https://github.com/Nuget/Home">open an issue on GitHub</a>.
+</div>
+
+
+###Who is this for?
+The beta channel is recommended for the following users:
+<ol>
+<li>1. You like to stay on the latest and greatest.</li>
+<li>2. You want to try out new features.</li>
+<li>3. You are experiencing a blocking issue and want to get access to the build with the fix before it hits RTM.</li>
+</ol>
+
+###How do I get access to the Beta Feed?
+You can get access to the Beta builds by the following the steps outlined below.
+
+<ol>
+<li>1. Add the Beta Feed: <b>https://dotnet.myget.org/F/nuget-beta/vsix/</b> to the Additional Extension Galleries list in<br> <b>Tools->Options->Environment->Extensions and Updates</b>.<br></li>
+<li><img src="../../images/BetaChannel/01_ToolsSettings.png"</img></li>
+
+<li>2. Navigate to <b>Tools->Extensions and Updates</b> and select <b>Online</b>. You should now be able to see the NuGet-Beta Feed there. Install the NuGet Package Manager Extension.<br></li>
+<li><img src="../../images/BetaChannel/02_ExtensionUpdate.png"</img></li>
+</ol>
+
+
 
-##Command-line Tools
-asdasdasdasdsd
 
 
 ##NuGet in .NET CLI
 asdasdasdasd
 
+##Package Explorer
+If you are <strong>GUI inclined</strong>, use the <a href="/Create/using-a-gui-to-build-packages" class="alert-link">Package Explorer GUI to create packages</a>.<br>
```

Comment:
Package Explorer is a community driver visual explorer and creator of nuget packages. GUI sometimes feels condescending